pandas/tests/test_multilevel.py

```diff
@@ -358,6 +358,53 @@ def test_unstack(self):
         # test that int32 work
         self.ymd.astype(np.int32).unstack()
 
+    def test_unstack_partial(self):
```

Comment:
ideally would like to parametrize this, have a look at pandas/tests/reshape/merge/test_merge.py for examples

Reply:
good point.  done!